lib/nodejs/worker.ex

```diff
@@ -56,15 +56,18 @@ defmodule NodeJS.Worker do
     {:ok, [node_service_path(), port]}
   end
 
-  defp get_response(data \\ '') do
+  defp get_response(data, timeout) do
     receive do
       {_, {:data, {flag, chunk}}} ->
         data = data ++ chunk
 
         case flag do
-          :noeol -> get_response(data)
-          :eol -> data
+          :noeol -> get_response(data, timeout)
+          :eol -> {:ok, data}
         end
+    after
+      timeout ->
```

Comment:
We don't need to use the pin operator is needed here?

Reply:
No, since it's not a pattern match.
I was also confused by this, as it's formatted like in a case statement. But the pin operator leads to a compiler error, doing it this way is correct.